numba/cuda/api.py

```diff
@@ -88,9 +88,9 @@ def is_cuda_array(obj):
 def is_float16_supported():
     """Whether 16-bit floats are supported.
 
-    Returns True if float16 is supported, False otherwise.
+    float16 is always supported in current versions of Numba - returns True.
```

Comment:
Can we leave this as-is? It seems simpler to think about for a user reading the docstring rather than thinking about what version of Numba is currently in use.

Reply:
I'm ok with that, thanks for considering it.